src/cli.ts

```diff
@@ -49,6 +49,7 @@ function _run () {
 
   function showUsage () {
     consola.info(`Usage: ${usage}`)
+    process.exit(1)
```

Comment:
Well I found it really ugly to have the process.exit(1).

![Screenshot 2020-06-16 at 13 27 31](https://user-images.githubusercontent.com/904724/84769002-7b50a900-afd5-11ea-94ad-ad93ed316f42.png)

Reply:
exit code 1 is standard for non-actions but will make it 0 then :)